plugins/profefe.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: profefe
+spec:
+  version: v0.8.0
+  homepage: https://github.com/profefe/kubectl-profefe
+  shortDescription: continuous profiling made easy in Kubernetes with profefe
+  description: |
```

Comment:
does the plugin come with profefe binary, or is it installed separately? if so, add `caveats:`.

Reply:
profefe is a registry, it is a separate `server` running somewhere. Should I add it as `caveats`?

Reply:
It wasn't clear to me this uploads profiles somewhere.
I assumed it directly downloads profiles from Kubernetes Pods.
That certainly sounds like a caveat.

but I saw two binaries in the tarball, I am not sure what's what.

Reply:
the second binary `kprofefe` is not a kubectl plugin, I just release them together for simplicity it is a cronjob. I changed the YAML in order to install only the plugin